README.md

```diff
@@ -1,157 +1,47 @@
-## Trashed
-# Keep an eye on resource usage.
+## Barnes - GC Statsd Reporter
 
-
- - Sends per-request object counts, heap growth, GC time, and more to StatsD.
- - Sends snapshots of resource usage, e.g. live String objects, to StatsD.
- - Supports new stuff: Rails 5.1 and latest Ruby 2.x features.
- - Supports old stuff: Rails 2/3/4, Ruby 1.9+, REE, Ruby 1.8 with RubyBench patches.
+A fork of [trashed](https://github.com/basecamp/trashed) focused on Ruby metrics for Heroku.
```

Comment:
Would add that it's *not* focused on per request metrics, but overall, whole process metrics.